docs/UserGuide.adoc

```diff
@@ -117,6 +119,10 @@ Examples:
 Returns `john` and `John Doe`
 * `find Betsy Tim John` +
 Returns any person having names `Betsy`, `Tim`, or `John`
+* `find p/999 555` +
+Returns any person having phone number `999` or `555`
+* `find p/999 e/test@example.com`
```

Comment:
Maybe can add in this example to user guide as well `find alex lidavid@example.com 87438807` which displays a list of people with any of their fields matching the keyword alex, lidavid@example.com or 87438807 (to show find searches all fields and not only names)